Paladin.cpp

```diff
@@ -1,7 +1,23 @@
 #include "Paladin.h"
+#include "Utility.h"
 
-// Paladin::Paladin
+Paladin::Paladin(std::string n,int hp, int armor) : Character(hp, armor, 10), name(n)
+{
+    helpfulItems = makeHelpfulItems(4);
+    defensiveItems = makeDefensiveItems(3);
+}
 
-// Paladin::getName
+const std::string& Paladin::getName()
+{
+    return name;
+}
 
-// Paladin::getStats
\ No newline at end of file
+void Paladin::attack(Character& other)
+{
+    Character::attack(other);
+}
```

Comment:
This function was not needed
it does nothing except call the base class's function
- [ ] remove it